src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -313,6 +313,21 @@ function warnAboutUnstableUse() {
   warned = true;
 }
 
+/**
+ * True if the supplied DOM node is a valid node element.
+ *
+ * @param {?DOMElement} node The candidate DOM node.
+ * @return {boolean} True if the DOM is a valid DOM node.
+ * @internal
+ */
+function isValidContainer(node) {
+  return !!(node && (
+    node.nodeType === ELEMENT_NODE_TYPE ||
+    node.nodeType === DOC_NODE_TYPE ||
+    node.nodeType === DOCUMENT_FRAGMENT_NODE_TYPE
+  ));
+}
+
```

Comment:
I just copy + pasted this from `ReactMount`, wasn't sure it was worth creating a new module, but I can if you want.